src/app/(tabs)/index.tsx

```
@@ -1,19 +1,23 @@
-import {StyleSheet} from "react-native";
-import {useTheme} from "@/theme";
-import SafeAreaView from '@/components/global/SafeAreaView'
-import Text from "@/components/global/Text";
+import {StyleSheet, View} from "react-native";
+import {MapViewComponent, RegionComponent} from "@/features/map/components";
 
 const Map = () => {
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)